src/overtime_analyzer/services/overtime_processor.py

```
@@ -0,0 +1,621 @@
+                    # 문자열 형식 처리 ("1:30" -> 1.5시간)
+                    time_str = row["초과근무시간"].replace(" ", "")
+                    if ":" in time_str:
+                        hours, minutes = map(int, time_str.split(":"))
+                        overtime_hours = hours + minutes / 60.0
+                    else:
+                        # 숫자 형식 문자열인 경우
+                        overtime_hours = float(time_str)
+                else:
+                    # 숫자 형식인 경우 직접 변환
+                    overtime_hours = float(row["초과근무시간"])
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)